<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" />
  <arg name="load_gripper" default="true" />
  <include file="$(find franka_control)/launch/franka_control.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>

  <rosparam command="load" file="$(find franka_more_controllers)/config/franka_more_controllers.yaml"  />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="gravity_compensation"/>
  <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find franka_example_controllers)/launch/robot.rviz"/>

  <arg name="record" default="false" />
  <group if="$(arg record)">
    		<node pkg="rosbag" type="record" name="record" output="screen" args="record -o /home/tran/catkin_ws/bag/  /joint_states /franka_state_controller/F_ext /franka_state_controller/EE_pose"/>
  </group>

</launch>
